doc/source/getting_started/about_odl.rst

```diff
@@ -63,12 +56,9 @@ or :math:`\mathbb{C}`), and :math:`n` a positive integer. Example:
 Function spaces
 ~~~~~~~~~~~~~~~
 
-A `function space`_ is a set of functions :math:`f: \mathcal{X} \to \mathcal{Y}` with fixed domain
-and range (more accurately: `codomain`_), where :math:`\mathcal{Y}` is a vector space. The ODL
-implementation `FunctionSpace` covers only the cases :math:`\mathcal{Y} = \mathbb{R}` or
-:math:`\mathbb{C}` since the general case has large overlaps with `Operator`. Note that we do
-not make a distinction between different types of function spaces with respect to regularity,
-integrability etc. on an *abstract* level since there is no obvious way to check it.
+A `function space`_ is a set of functions :math:`f: \mathcal{X} \to \mathcal{Y}` with fixed domain and range (more accurately: `codomain`_), where :math:`\mathcal{Y}` is a vector space.
+The ODL implementation `FunctionSpace` covers only the cases :math:`\mathcal{Y} = \mathbb{R}` or :math:`\mathbb{C}` since the general case has large overlaps with `Operator`.
+Note that we do not make a distinction between different types of function spaces with respect to regularity, integrability etc. on an *abstract* level since there is no obvious way to check it.
 
 As linear spaces, function spaces support some interesting operations:
```

Comment:
While reading the example i found this line:

>>> ratio_func = exp_plus_one / exp  # x -> exp(x) / (exp(x) + 1)

The comment has to be a misstake, right?